ui/app/src/components/confirmationDialog/RemoveAnimatedConfirmation.vue

```diff
@@ -0,0 +1,129 @@
+<template>
```

Comment:
We really need to DRY these conf dialogs out they slowed me down a heap trying to get ready for the demo